static/src/js/components/Preferences/RadioField.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { startCase } from 'lodash'
+
+class RadioField extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      formData: props.formData
+    }
+
+    this.onChange = this.onChange.bind(this)
+  }
+
+  onChange() {
+    const { onChange } = this.props
+
+    return (event) => {
+      this.setState({
+        formData: event.target.value
+      }, () => {
+        const { formData } = this.state
+        return onChange(formData)
+      })
+    }
+  }
+
+  render() {
+    const {
+      name: fieldName,
+      formData,
+      schema
+    } = this.props
+
+    const {
+      enum: values,
+      enumNames,
+      description
+    } = schema
+
+    return (
+      <div className="preferences-form__radio">
```

Comment:
For consistency, I think we should use `react-bootstrap`s grid and form components here